CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 0.1.1
+- Fix pub score.
```

Comment:
- Address pub score: follow dart File conventions, provide documentation, and pass static analysis.